### ISSUE TITLE
No ParseException when column starts with number

### ISSUE THREAD

**Issue by Belzee01** [JSQLParser/JSqlParser#1147]
**Describe the bug**
Currently when parsing query columns starting with number are ignored.

**To Reproduce**
1. Example query:
SELECT abc.1table FROM test
would result in one element list from net.sf.jsqlparser.parser.CCJSqlParser#RelObjectNameList
data: [ "abc" ] 
where "1table" name is ignored due to 
`if (jj_2_28(2)) {
        ;
      } else {
        break label_34;
      }`
where method jj_2_28(2) returns false because jj_scanpos.kind 270 != 260 identifier. 

**Expected behavior**
I would assume that this check throw ParseException when identifier starts with number.

**System**
- Java Version 11
- JSqlParser version 4.0

**Comment by wumpz** [comment]
That is right. JSqlParser does not support column names that start with a number. You have to quote it:

```sql
SELECT abc."1table" FROM test
```
